lib/utils.js

```diff
@@ -77,6 +77,15 @@ export function expandTabs(line) {
     });
 }
 
+export function maskRootdir(filepath) {
+    if (filepath) {
+        // todo: make this compatible with local installations and windows etc
+        return filepath.replace(/^\/tmp\/compiler-explorer-compiler[\w\d-.]*\//, '/app/').replace(/^\/app\//, '');
```

Comment:
I wondered how the tests changed (all the previously-coded `/tmp/compiler-explorer-compiler//opt/blah` references becoming `/opt/blah`). A neat approach, though obviously (like you note) dependent on the installation and some specifics of the `newTempDir` function.

Can you perhaps extract the common "must be the same" things between this and `newTempDir()` in base-compiler (maybe `newTempDir()` could move to `utils`, then they can share the `compiler-explorer-compiler` string _and_ the `process.env.tmpDir` which might actually "fix" the TODO above!